src/utils/allocator_n64.h

```diff
@@ -0,0 +1,65 @@
+#ifndef ALLOCATOR_DEFAULT_H
+#define ALLOCATOR_DEFAULT_H
+
+#include <assert.h>
+#include <stddef.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+// Allocator options
+#define ALLOC_HINT_TEXTURE 0x00000001
```

Comment:
unrelated to the bitflags conversation: If `ALLOC_HINT_TEXTURE` is to be used in cross-platform code, it should not be defined here, but in allocator.h.